sys/src/main/java/com/alibaba/nacos/sys/env/EnvUtil.java

```diff
@@ -361,19 +361,34 @@ public static String getMemberList() {
     private static final String FILE_PREFIX = "file:";
     
     public static Resource getApplicationConfFileResource() {
+        Resource customResource = getCustomFileResource();
+        if (customResource == null) {
```

Comment:
Simply to `null == customResource ? return getDefaultResource() : customResource`